src/init-reference.js

```diff
@@ -82,3 +82,19 @@ registerWidget('deck-comment', (el, { richObjectType, richObject, accessible })
 		},
 	}).$mount(el)
 })
+
+registerCustomPickerElement('create-new-deck-card', async (el, { providerId, accessible }) => {
```

Comment:
Let's also pass `normal` for the size here to match the requirements from https://github.com/nextcloud/deck/issues/4962#issuecomment-1661721341 

Reference: https://github.com/nextcloud-libraries/nextcloud-vue/blob/8dbc4451dfef7b0edf7ed438f26b2064ffddd113/src/components/NcRichText/NcReferencePicker/customPickerElements.js#L28